pyslang/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
I think all of these auxiliary files can be left out since they exist in the parent directory:
.editorconfig
.flake8
.gitattributes
.gitignore
.gitmodules
.pre-commit-config